src/goto-symex/symex_target_equation.cpp

```diff
@@ -1026,7 +1026,7 @@ irep_idt symex_target_equationt::SSA_stept::get_property_id() const
   }
   else
   {
-    // return empty
+    UNREACHABLE;
```

Comment:
Just a comment, no action required: If we want to introduce unwinding assertions for recursions then there would be another case here.

Reply:
Err, we do. I'll fix the code and will add a test.

Reply:
(Done)